FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -195,40 +196,41 @@ def loadDiracCfg( self, verbose = False ):
     self.mysqlRootPwd = self.localCfg.getOption( cfgInstallPath( 'Database', 'RootPwd' ), self.mysqlRootPwd )
     if verbose and self.mysqlRootPwd:
       gLogger.notice( 'Reading Root MySQL Password from local configuration' )
+    else:
+      gLogger.warn( 'MySQL root password not found' )
 
-    self.mysqlUser = self.localCfg.getOption( cfgInstallPath( 'Database', 'User' ), '' )
-    if self.mysqlUser:
-      if verbose:
-        gLogger.notice( 'Reading MySQL User from local configuration' )
+    self.mysqlUser = self.localCfg.getOption( cfgInstallPath( 'Database', 'User' ), self.mysqlUser )
+    if verbose and self.mysqlUser:
```

Comment:
I don't understand why do we use this verbose flag instead of setting the gLogger to an appropriate level ?